build/config/ios/ios_sdk.py

```diff
@@ -99,7 +104,16 @@ def main(argv):
       sdk,
       'Path'
     ]
-    sdk_output = run_command_with_retry(command, timeout=300)
+    if args.xcode is not None:
+      if sdk in ['iphoneos']:
+        sdk_output = os.path.join(args.xcode, 'iPhoneOS.platform/Developer/SDKs/iPhoneOS17.0.sdk')
```

Comment:
I really don't want the version hardcoded since this is coming from the Xcode sdk version in the .ci.yaml.

Can we try swapping the ios_sdk and mac_sdk python script to `xcrun --sdk <sdk> --show-sdk-path` instead, and see if it still times out?

Reply:
I agree, and I'll try that, yeah.